integration_tests/array_size_02.py

```diff
@@ -0,0 +1,90 @@
+from lpython import i32, f64, c32, c64, u32
+from numpy import empty, size
+
+def main0():
+    x: i32[4, 5, 2] = empty([4, 5, 2])
+    y: f64[24, 100, 2, 5] = empty([24, 100, 2, 5])
```

Comment:
I noticed one problem here, but I created a dedicated issue for this: https://github.com/lcompilers/lpython/issues/1790, as we need to test that LPython reports an error here.